include/net/lwm2m.h

```diff
@@ -45,6 +45,7 @@
 #define LWM2M_OBJECT_LOCATION_ID                6
 #define LWM2M_OBJECT_CONNECTIVITY_STATISTICS_ID 7
 #define LWM2M_OBJECT_SOFTWARE_MANAGEMENT_ID     9
+#define LWM2M_OBJECT_PORTFOLIO_ID		16
```

Comment:
tabs are for indentation, not sub-alignment like here.

Reply:
Ok, I will fix that.